README.md

```diff
@@ -63,3 +68,15 @@ Des blagues et des animaux. Proposer la vôtre si elle me faire rire, je l'inclu
 
 **R:** Le raton-loveur.
 
+---
+
+# In Persian 🇮🇷
+
+---
+
+**Q:** چرا ماهی‌ها هیچ‌وقت اینترنت ندارند
+
+**A:** چون از شبکه‌ها می‌ترسند     
```

Comment:
A and Q stands for Question and answer, please use only the corresponding acronyms in Persian :)